### ISSUE TITLE
webpack 4 crash

### ISSUE THREAD

**Issue by budarin** [gajus/prepack-webpack-plugin#33]
I have include plugin in my config and got the error:
```
....
[0] finish module graph
  0% [0] after dependencies optimization
[0] advanced module optimization
[0] basic chunk optimization
[0] basic chunk optimization
[0] advanced chunk optimization
[0] basic chunk optimization
[0] chunk modules optimization
  0% [0] after chunk modules optimization
[0] chunk reviving
  0% [0] hashing
  0% [0] chunk assets processing
  0% [0] additional chunk assets processing
[0] chunk asset optimization
Warn: PP1023: Program may terminate with exception: not an object
Error
  TypeError
    
    at e79af574d74c913e5be8.chunk.js:2:35 at 2:34 to 2:48
Warn: PP1023: Program may terminate with exception: not an object

.......  here goes a list of all chunks with such error
    
...\node_modules\prepack-webpack-plugin\dist\PrepackPlugin.js:27
  return chunk.hasRuntime() && chunk.isInitial();
                                     ^

TypeError: chunk.isInitial is not a function
    at isEntryChunk (...\node_modules\prepack-webpack-plugin\src\PrepackPlugin.js:23:38)
    at compilation.plugin (...\node_modules\prepack-webpack-plugin\src\PrepackPlugin.js:56:26)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (...\node_modules\tapable\lib\HookCodeFactory.js:24:12), <anonymous>:12:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (...\node_modules\tapable\lib\Hook.js:35:21)
    at hooks.additionalAssets.callAsync.err (...\node_modules\webpack\lib\Compilation.js:943:36)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (...\node_modules\tapable\lib\HookCodeFactory.js:24:12), <anonymous>:9:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (...\node_modules\tapable\lib\Hook.js:35:21)
    at hooks.optimizeTree.callAsync.err (...\node_modules\webpack\lib\Compilation.js:939:32)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (...\node_modules\tapable\lib\HookCodeFactory.js:24:12), <anonymous>:9:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (...\node_modules\tapable\lib\Hook.js:35:21)
    at Compilation.seal (...\node_modules\webpack\lib\Compilation.js:881:27)
    at hooks.make.callAsync.err (...\node_modules\webpack\lib\Compiler.js:464:17)
    at _err0 (eval at create (...\node_modules\tapable\lib\HookCodeFactory.js:24:12), <anonymous>:11:1)
    at _addModuleChain (...\node_modules\webpack\lib\Compilation.js:749:12)
    at processModuleDependencies.err (...\node_modules\webpack\lib\Compilation.js:688:9)
    at process._tickCallback (internal/process/next_tick.js:150:11)
script exit code: 1npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! kometa@1.0.0 build: `prun build`
npm ERR! Exit status 1
npm ERR! 
```
here is my part of optimization config
```
optimization: {
        splitChunks: {
            cacheGroups: {
                vendor: {
                    test: /node_modules/, // you may add "vendor.js" here if you want to
                    name: 'vendor',
                    chunks: 'initial',
                    enforce: true,
                },
            },
        },
        runtimeChunk: {
            name: 'runtime',
        },
        minimizer: [new BabiliPlugin(), new OptimizeJsPlugin({ sourceMap: false })],
    },
```

**Comment by bazaglia** [comment]
Same here. `TypeError: chunk.isInitial is not a function`.

**Comment by gajus** [comment]
It is open-source package. I don't have resources to inspect this issue at the moment. However, PR would be welcome.

**Comment by stackdumper** [comment]
Fixed in `1.1.1` version
